src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -136,9 +136,15 @@ include the `src` and a `sizes` property. Unlike [web app manifest icons], the
 
 SVG files are not supported at the time of writing, use PNG instead.
 
-If you want pixel-perfect icons, provide them in
-increments of 48dp. Otherwise, it is recommended that you use a 192x192 pixels
-icon.
+If you want pixel-perfect icons, provide them in increments of 48dp. Otherwise,
+it is recommended that you use a 192x192 pixels icon.
+
+{% Aside %}
+As a quality measure, icons must be at least half of the device's ideal size
+(48dp) on Android. This means the shortcut icon won't be visible on xxhdpi
+screens for instance if a 72x72 pixels icon is not included (`48 x (480 / 160) /
+2 = 72`).
```

Comment:
I don't understand this sentence.

Reply:
Which part?

Reply:
Shall I link to an article explaining xxhdpi displays?

Reply:
The icon being invisible if it's too small seems like part of the main point, but you've attached it to the example. If 48 is the device's ideal, why does 72x72 produce no result?

Reply:
Dp and pixels are two different things.